spec/erb_lint/linters/rubocop_spec.rb

```diff
@@ -161,7 +218,7 @@
     context 'correct alignment with html preceeding erb' do
       let(:file) { <<~FILE }
         <div><a><br><% ui_helper :foo,
-                      checked: true %>
+                         checked: true %>
```

Comment:
While playing with auto-correction I realized aligning the code on `<%` does not produce expected results when aligning hash or array elements that are part of a method call. Now the ruby code will be aligned exactly as it is in the html file, the html up to and including `<%=` is replaced with spaces so everything aligns correctly.